spec/sword/endpoints/proquest_endpoint_spec.rb

```diff
@@ -35,10 +35,6 @@
         # for test purposes, turn off actual sending of POST request
         @proquest_endpoint.hyacinth_adapter.no_op_post = true
       end
-
-      it '#handle_deposit is called successfully (without crashing)' do
```

Comment:
Based on the other changes in this PR, it's not clear to me why this test was removed.

Reply:
This test is brittle and not very useful. If required, I can create a ticket to add the test to a future release.

Reply:
See: https://github.com/cul/ldpd-sword/pull/14/files#r1826322269